src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioOperationServiceImpl.java

```diff
@@ -225,4 +266,57 @@ private Path generateOutputPath(Path basePath,
         return basePath.resolve(filename);
     }
 
+    private List<List<AudioFileInfo>> getSeparatedAudioFileInfo(List<AudioFileInfo> audioFileInfoList, int acceptableAudioGapSecs) {
+
+        List<List<AudioFileInfo>> seperatedAudioFileInfoList = new ArrayList<>();
+
+        Iterator<AudioFileInfo> firstAudioFileInfoIterator = audioFileInfoList.iterator();
+        Iterator<AudioFileInfo> secondAudioFileInfoIterator = audioFileInfoList.iterator();
+
+        AudioFileInfo firstAudioFileInfo;
+        AudioFileInfo secondAudioFileInfo = getNextAudioFileInfo(secondAudioFileInfoIterator);
```

Comment:
initialized so that iterator gets advanced to correct place - i've added comment to code so it makes more sense:

        // initialized here to advance iterator
        AudioFileInfo secondAudioFileInfo = getNextAudioFileInfo(secondAudioFileInfoIterator);

Reply:
removed now - as per later comment